app/controllers/api/projects.php

```diff
@@ -1385,11 +1454,10 @@
             throw new Exception('Failed to verify domain', 401);
         }
 
-        $project->findAndReplace('$id', $domain->getId(), $domain
-                ->setAttribute('verification', true)
-            , 'domains');
+        $domain->setAttribute('verification', true);
 
-        $dbForConsole->updateDocument('projects', $project->getId(), $project);
+        $dbForConsole->updateDocument('domains', $domain->getId(), $domain);
```

Comment:
Can condense this down a bit since `$document->setAttribute()` returns `self`
```suggestion
        $dbForConsole->updateDocument('domains', $domain->getId(), $domain->setAttribute('verification', true));
```